specification/eventhub/resource-manager/Microsoft.EventHub/preview/2021-06-01-preview/eventhubs.json

```diff
@@ -248,15 +248,11 @@
         "intervalInSeconds": {
           "format": "int32",
           "type": "integer",
-          "maximum": 900,
-          "minimum": 60,
           "description": "The time window allows you to set the frequency with which the capture to Azure Blobs will happen, value should between 60 to 900 seconds"
         },
         "sizeLimitInBytes": {
           "format": "int32",
           "type": "integer",
-          "maximum": 524288000,
-          "minimum": 10485760,
           "description": "The size window defines the amount of data built up in your Event Hub before an capture operation, value should be between 10485760 to 524288000 bytes"
         },
```

Comment:
@v-Ajnava The description also needs to be either removed or changed

Reply:
as recommended, removed  min/max and description will have the range.